tensorflow_graphics/datasets/features/__init__.py

```diff
@@ -15,7 +15,13 @@
 """`tensorflow_graphics.datasets.features` API defining feature types."""
 
 from tensorflow_graphics.datasets.features.trimesh_feature import TriangleMesh
```

Comment:
Change import order to:

from tensorflow_graphics.datasets.features.camera_feature import Camera
from tensorflow_graphics.datasets.features.pose_feature import Pose
from tensorflow_graphics.datasets.features.trimesh_feature import TriangleMesh
from tensorflow_graphics.datasets.features.voxel_feature import VoxelGrid